text/0000-deprecate-authors-field.md

```diff
@@ -0,0 +1,128 @@
+# RFC: Deprecate the authors field
+
+- Feature Name: `deprecate_authors_field`
+- Start Date: 2021-01-07
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes to deprecate the `package.authors` field of `Cargo.toml`.
+This also implies preventing Cargo from auto-filling it, allowing crates to be
+published to crates.io without the field being present, and avoiding displaying
+its contents on the crates.io and docs.rs UI.
+
+# Motivation
+[motivation]: #motivation
+
+The crates.io registry does not allow users to change the contents of already
+published versions: this is highly desirable to ensure working builds don't
+break in the future, but it also has the unfortunate side-effect of preventing
+people from updating the list of crate authors defined in `Cargo.toml`'s
+`package.authors` field.
+
+This is especially problematic when people change their name or want to remove
+their name from the Internet, and the crates.io team doesn't have any way to
+address that at the moment except for deleting the affected crates or versions
+altogether. We don't do that lightly, but there were a few cases where we were
+forced to do so.
+
+The contents of the field also tend to scale poorly as the size of a project
+grows, with projects either making the field useless by just stating "The
+$PROJECT developers" or only naming the original authors without mentioning
+other major contributors.
```

Comment:
What if we look at it from another way? Authors is not for accreditation, but for contacting a maintainer. In that case what if we just rename `authors` to `maintainer`/`contact`?

Reply:
That's not the main reason why I'd like for this RFC to land. It's another effect that I personally think is positive, but it's more of a collateral benefit.

Reply:
If the authors want to be contactable they can provide contact details in the description/readme/homepage still (I assume most maintainers will want to be contacted via their projects issue tracker, not random emails).

Reply:
If `authors` is for contact information, you've got the same problem again – people's contact info changes in a manner completely unrelated to crate versions. Do you make a minor release when you change your email address? Things like this shouldn't even need to be in the version control, imo, because they're conceptually unlinked from the software.